bin/edit.js

```diff
@@ -26,6 +35,10 @@ function handleEdit(args) {
     var wtName  = args.name ? args.name + '/' : '';
     var url     = profile.url + '/edit/' + profile.container + '#/' + wtName + profile.token;
 
+    if (args.previousVersion) {
+        url = profile.url + '/prev/edit/' + profile.container + '#/' + wtName + profile.token;
```

Comment:
We can change `--previous-version` to `--version v1`/ `--version v2`. Only `major` should be supported since the webtask-runtime is pointing to the `major` and only there could be breaking changes on major changes.